Updated Step 7 of "Provision the simulated device"

abhipsaMisra opened the pull request:
Step 7 under "Provision the simulated device" is unclear regarding the Enter command. Added an extended explanation.

azurecla commented:
Hi __@ostrich75__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (larryzh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ostrich75 commented:
#sign-off

ostrich75 commented:
#sign-off

azurecla commented:
Hi __@ostrich75__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (larryzh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ostrich75 commented:
#sign-off

abhipsaMisra committed:
Updated Step 7 of "Provision the simulated device"

Step 7 under "Provision the simulated device" is unclear regarding the Enter command. Added an extended explanation.
